tests/posit/connect/test_content.py

```diff
@@ -6,6 +6,28 @@
 from .api import load_mock  # type: ignore
 
 
+class TestContent:
+    @responses.activate
+    def test_update(self):
+        guid = "f2f37341-e21d-3d80-c698-a935ad614066"
+        responses.get(
+            f"https://connect.example/__api__/v1/content/{guid}",
+            json=load_mock(f"v1/content/{guid}.json"),
+        )
+        con = Client("12345", "https://connect.example")
+        content = con.content.get(guid)
+        assert content.guid == guid
+
+        new_name = "New Name"
+        responses.patch(
```

Comment:
So this is PATCH but user is PUT? 🤦

Reply:
Yep!

Reply:
Good thing we're writing an SDK to encapsulate that!